infrastructure/lambda-bulk-upload-report.tf

```diff
@@ -0,0 +1,23 @@
+module "bulk-upload-report-lambda" {
+  source  = "./modules/lambda"
+  name    = "BulkUploadReportLambda"
+  handler = "handlers.bulk_upload_report_handler.lambda_handler"
+  iam_role_policies = [
+    "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole",
+    "arn:aws:iam::aws:policy/CloudWatchLambdaInsightsExecutionRolePolicy",
+    module.ndr-bulk-staging-store.s3_object_access_policy,
+  ]
```

Comment:
Does this lambda need to read from bulk_upload_dynamodb_table?   If yes then maybe add a role policy to read from bulk_upload_dynamodb_table